### ISSUE TITLE
add support for TP-LINK ArcherC50v4

### ISSUE THREAD

**Issue by Akira25** [freifunk-berlin/firmware#643]
It seems that the C50v3 is not shipped anymore. Instead C50v4 is shipped.
Hardware-wise they are identical. The only difference is the flash layout. The bootloader is split into to parts.
Support in OpenWRT was added with this commit https://github.com/blocktrron/openwrt/commit/2410f411f3bc32d6110d1df34a876fff181706b9. There are further details given.
Could you please configure the buildbot to build that image to?

**Comment by SvenRoederer** [comment]
It's not merged into openwrt master (yet). 
can you provide a patch, that we can include now?

**Comment by SvenRoederer** [comment]
These days the floowing patches have been sent to the OpenWrt ML:
* http://lists.infradead.org/pipermail/openwrt-devel/2018-December/015295.html
* http://lists.infradead.org/pipermail/openwrt-devel/2018-December/015294.html

probably the support will added upstream within the next days.
Feel free to send a PR towards our firmware if you want to have the support urgently, or we just wait on upstream

**Comment by SvenRoederer** [comment]
this board ist now supported on OpenWrt-master, we added it in https://github.com/freifunk-berlin/firmware/commit/9c5d51ba60c26b53b07324a584c73583e162e960

**Comment by SvenRoederer** [comment]
OpenWrt-master is used by our branches "daily/upstream-master" and "SAm0815_experimental"
